routers/api/v1/notify/repo.go

```diff
@@ -65,7 +65,7 @@ func ListRepoNotifications(ctx *context.APIContext) {
 	// - name: all
 	//   in: query
 	//   description: If true, show notifications marked as read. Default value is false
-	//   type: string
+	//   type: boolean
```

Comment:
It's unrelated changes?

Reply:
yes see https://github.com/go-gitea/gitea/pull/16168#issuecomment-862629624